uwblueprint/headsupguys
# Settings

## Notion ticket link

[Settings Page](https://www.notion.so/uwblueprintexecs/Settings-Page-7b55ff46d7c04a26a1320ec5cdd28be5)
[Figma](https://www.figma.com/file/GhDd9C0XsyVDgwL5zzvDvO/HUG-S21?node-id=856%3A10017)



## Implementation description
* create the settings page with a table of admin users
* create 3 modals (ADD, EDIT, DELETE)
* create `DropDown` component with label and error message
* used `console.log` to mock sending requests to the back end
note: Chakra provides some defined sizes for Modals so the size doesn't exactly match the design


![image](https://user-images.githubusercontent.com/42286268/127785138-59943be1-f4af-4aff-8cc5-d4c79cdb016d.png)
![image](https://user-images.githubusercontent.com/42286268/127785150-8f8c5def-d901-483a-82fc-4433ef68debc.png)
![image](https://user-images.githubusercontent.com/42286268/127785172-b8a1361e-e94f-40ab-a266-ff88f794e5b4.png)
![image](https://user-images.githubusercontent.com/42286268/127785179-b63b476d-a81b-481e-8701-65e8daa175dd.png)
![image](https://user-images.githubusercontent.com/42286268/127785243-fb2a69f3-1519-4e3d-afba-7d7b5cc191eb.png)
![image](https://user-images.githubusercontent.com/42286268/128276065-3254284a-30e5-46ad-a0ea-fa475f03f70e.png)

## Steps to test
1. run locally and go to `localhost:3000/admin/settings`
2. open chrome inspect
3. add, edit, delete users and see ensure it's reflected in the console
4. try to add or edit a user with a blank/invalid email, ensure an error appears


## What should reviewers focus on?
* the modals reflect the correct user


## Checklist
